testdata/r8/deobfuscator/1/de-obfuscated-crash.txt

```diff
@@ -0,0 +1,22 @@
+E/AndroidRuntime: FATAL EXCEPTION: main
```

Comment:
I wonder if it will pass windows CI if we remove `.txt` extension from de-obfuscated-crash.txt and obfuscated-crash.txt. I believe it will stop git cloning the txt with crlf and fix the tests.

Reply:
I've added those changes, let's wait to see if that works!

Reply:
I managed to fix it by forcing LF line endings on all platforms. See https://github.com/elastic/apm-server/pull/10545/commits/9516803085a614a4be95608d74333a138b84fd3d